operator/src/main/java/org/bf2/operator/controllers/ManagedKafkaController.java

```diff
@@ -214,6 +217,11 @@ private void updateManagedKafkaStatus(ManagedKafka managedKafka, Optional<Operan
             status.setAdminServerURI(kafkaInstance.getAdminServer().uri(managedKafka));
             status.setServiceAccounts(managedKafka.getSpec().getServiceAccounts());
         }
+
+        if (!Objects.equals(originalStatus, status)) {
+            status.setUpdatedTimestamp(ConditionUtils.iso8601Now());
```

Comment:
@machi1990 a quick search of the fleetmanager code base suggests that the managed kafka status updated timestamp is not being used.  Is this something that could be removed in another JIRA?

Reply:
You are correct. We do not use nor have it in the status schema: https://github.com/bf2fc6cc711aee1a0c2a/kas-fleet-manager/blob/009aca3b4daff751d26bade6827cdac66bf64e0d/openapi/kas-fleet-manager-private.yaml#L403-L470

Was it meant to be used in some cases? I don't seem to recall if we've something lined up that will make use of it so +1 on removing it.

Reply:
Tracking the removal of the timestamp in MGDSTRM-10096